src/Security/Authentication/JwtBearer/src/JwtBearerOptions.cs

```diff
@@ -15,13 +16,17 @@ namespace Microsoft.AspNetCore.Authentication.JwtBearer;
 public class JwtBearerOptions : AuthenticationSchemeOptions
 {
     private readonly JwtSecurityTokenHandler _defaultHandler = new JwtSecurityTokenHandler();
+    private readonly JsonWebTokenHandler _defaultTokenHandler = new JsonWebTokenHandler();
+    private bool _mapInboundClaims = JwtSecurityTokenHandler.DefaultMapInboundClaims;
 
     /// <summary>
     /// Initializes a new instance of <see cref="JwtBearerOptions"/>.
     /// </summary>
     public JwtBearerOptions()
     {
         SecurityTokenValidators = new List<ISecurityTokenValidator> { _defaultHandler };
+        // TODO - communicate to IdentityModel team to see if ITokenValidator interface can be created.
+        TokenHandlers = new List<TokenHandler> { _defaultTokenHandler };
```

Comment:
@kevinchalet i am thinking that creating the interface ITokenValidator that has a couple of methods instead of the abstract class TokenHandler, this would make it easier for users who currently have an implementation of ISecurityTokenValidator.

Reply:
> Thoughts?

IM already has `TokenHandler`, `SecurityTokenHandler` and `ISecurityTokenValidator`, which makes using these abstractions already quite painful. I'm not sure adding a 4th one will help 😄 

WIF only had `SecurityTokenHandler` and it was much clearer: unifying everything in the next Wilson major version would be more than welcome 😄

Reply:
@kevinchalet OK let's stick with TokenHandler then.

Reply:
@kevinchalet we could implement ISecurityTokenValidator on JsonWebTokenHandler, but just because we can cast it to TokenHandler doesn't mean that users will not have a runtime break as the SecurityToken will change from JwtSecurityToken to JsonWebToken.

Reply:
Sure, but that would mean an instance of `JsonWebTokenHandler` was added to `SecurityTokenValidators`, which, if you keep making using the new JWT stack opt-in, requires a deliberate action from the developer. And in this case, getting a `JsonWebToken` instead of a `JwtSecurityToken` is a logical result.

Reply:
I'm hoping to make it opt-out.

Reply:
Addressed: 507d24137b37c43ddf9c18bdeb7510e70c84bcf1

@Tratcher default is true, use TokenHandlers.